src/SprykerEco/Zed/Payone/Business/PayoneFacade.php

```diff
@@ -52,7 +57,7 @@ class PayoneFacade extends AbstractFacade implements PayoneFacadeInterface
      *
      * @return void
      */
-    public function saveOrder(QuoteTransfer $quoteTransfer, CheckoutResponseTransfer $checkoutResponse)
+    public function saveOrder(QuoteTransfer $quoteTransfer, CheckoutResponseTransfer $checkoutResponse): void
```

Comment:
It's a part of public API, please, remove added typehints.

Reply:
Fixed.